static/app/views/insights/agentMonitoring/components/llmCosts.tsx

```diff
@@ -15,7 +15,7 @@ export function LLMCosts({cost, className}: LLMCostsProps) {
         maximumFractionDigits: 8,
       })}
     >
-      {formatLLMCosts(cost)}
+      {cost ? formatLLMCosts(cost) : '-'}
```

Comment:
nit: at one point we cared a lot about using either `'\u2014'` or `&mdash;` instead of just `'-'`